docs/core/tools/local-tools-how-to-use.md

```diff
@@ -6,7 +6,7 @@ ms.date: 02/12/2020
 
 # Tutorial: Install and use a .NET Core local tool using the .NET Core CLI
 
-**This article applies to:** ✔️ .NET Core 2.1 SDK and later versions
+**This article applies to:** ✔️ .NET Core 3.0 SDK and later versions
```

Comment:
Prereqs need changing too

Reply:
That's covered by the "complete the first tutorial" prereq, as the first tutorial requires 3.1.  I had the applies to line on tutorial 1 as 3.0 but changed it since global tools apply to 2.1.  I'll add some text to the prereq there to explain the mismatch.

Reply:
Ahh didn't notice that the tool installed uses 2.1,